### DISCUSSION TITLE
All LoRAs showing in model checkpoint list

### DISCUSSION THREAD

**Discussion by dfjunk** [vladmandic/sdnext#1057] [Q&A]
I've done a lot of searching here and in the settings myself to figure this out, but all of my LoRAs seem to show up in the model checkpoint list and I can't figure out why. I've double checked and all my files are in the appropriate folders and there doesn't seem to be any settings that would allow them to all show up in there. Any ideas?

**Comment by vladmandic** [comment]
i've fixed that already.

**Reply by dfjunk** [reply]
Thanks for the reply, and apologies if I missed something. I installed this for the very first time this morning. Was it a recent fix since then, or did I miss a step in the installation process? I haven't enabled any arguments to avoid updates so I think I should be completely up to date. 

Thanks again

**Comment by vladmandic** [comment]
it was fixed 1.5h ago, you need version `df1fae7` or latest `8b68218`

**Reply by dfjunk** [reply]
Great, thanks! I'll make sure I'm up to date then. Just bad timing on my part then.
